proto/buildbuddy_service.proto

```diff
@@ -32,6 +32,8 @@ service BuildBuddyService {
       returns (invocation.UpdateInvocationResponse);
   rpc DeleteInvocation(invocation.DeleteInvocationRequest)
       returns (invocation.DeleteInvocationResponse);
+  rpc CancelInvocation(invocation.CancelInvocationRequest)
```

Comment:
I think we would be better off calling this `CancelExecutions` as this API does not affect the invocation directly.

Reply:
Done